teuthology/repo_utils.py

```diff
@@ -97,20 +97,25 @@ def enforce_repo_state(repo_url, dest_path, branch, remove_on_error=True):
         raise
 
 
-def clone_repo(repo_url, dest_path, branch):
+def clone_repo(repo_url, dest_path, branch, shallow=True):
```

Comment:
It's all inside repo_utils, but it gets used for teuthology and ceph-qa-suite both.  The other one I saw was in the cephfs tests in ceph-qa-suite, but I see now that's a locally-defined routine of the same name.

Mostly I was raising the question to make certain you'd doublechecked all the call paths.  I can't think of why teuthology would care that it's depth 1, but it's a change.